Sources/Fullscreen/AdConfirmationView/Views/AdConfirmationObjectView.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+public class AdConfirmationObjectView: UIView {
+    private lazy var imageView: RemoteImageView = {
+        let imageView = RemoteImageView(withAutoLayout: true)
+        imageView.dataSource = self
+        imageView.clipsToBounds = true
+        imageView.layer.cornerRadius = 8.0
+        return imageView
+    }()
+
+    private lazy var titleLabel: Label = {
+        let titleLabel = Label(style: .title3Strong, withAutoLayout: true)
+        titleLabel.numberOfLines = 0
+        titleLabel.textAlignment = .center
+    return titleLabel
+    }()
+
+    private lazy var bodyLabel: Label = {
+        let bodyLabel = Label(style: .body, withAutoLayout: true)
+        bodyLabel.numberOfLines = 0
+        bodyLabel.textAlignment = .center
+        return bodyLabel
+    }()
+
+    private let imageSize: CGFloat = 88.0
+    private let fallbackImageSize: CGFloat = 40.0
+
+    private var imageTopAnchorConstraint: NSLayoutConstraint?
+    private var imageWidthConstraint: NSLayoutConstraint?
+    private var imageHeightConstraint: NSLayoutConstraint?
```

Comment:
These three could be `lazy var`s, so you don't need to create and have the `guard` in `setup()`

Reply:
Alternatively: 
`[controversial opinion]` it would actually be OK to force unwrap these in setup(), since they were literally just created.`[/controversial opinion]` 😬